server/schema/order/res.js

```diff
@@ -48,10 +35,14 @@ const resolvers = {
           .transaction(async t => {
             await Order.create(input, {transaction: t}).then(async ({id}) => {
               orderId = id
+              let menuIds = input.orderDetails.map( orderDetail =>{
+                return orderDetail.menuId
+              })
+              let menus = await Menu.findAll({where: {id: menuIds}})
               await Promise.all(
                 input.orderDetails.map(async orderDetail => {
                   orderDetail.orderId = id
-                  orderDetail.price = await calcOrderDetailPrice(orderDetail)
+                  orderDetail.price = await calcOrderDetailPrice(orderDetail, menus)
```

Comment:
should pass in "menuPrice" instead of "menus" to this function calcOrderDetailPrice